kv/txn_test.go

```diff
@@ -478,3 +478,172 @@ func TestTxnTimestampRegression(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// When the gap between two writes in a txn is bigger than 10 seconds,
+// lowWater in timestamp cache can increase to the timestamp of the
+// first write as side effect of read in another txn, but when the txn
+// committed, the write's timestamp is not increased. so another txn
+// two reads before and after the second write will get different
+// value.
+func TestTxnRepeatGetDifferentValue(t *testing.T) {
```

Comment:
Now that the bug has been fixed, this test should be renamed to TestTxnRepeatGetSameValue and the comment updated accordingly.